### PR TITLE
[Feat] Add privacy modal

### LINKED ISSUE #825: Add a "privacy" pop up message upon logging into the application
**Is your feature request related to a problem? Please describe.**
In I-Talent, employees will be providing information that is considered "personal" such as education.  It's important that they confirm they understand the legistlation that applies and how the information will be used. 
 
**The solution**
Everytime a user logs in to I-Talent they should be prompted with a Message (with a privacy notice statement)
They will be required to confirm by clicking on an "ok" button.

Text for the statement in both languages to follow. 

image

### PR DESCRIPTION
**Pull request by dynamic11**
#### Changes introduced
- create privacy modal that shows every time on login
- if user accepts the "terms" they may continue and if they decline they are automatically logged out

#### Related issue(s)
closes #825

#### Screenshots (if applicable)
image

#### Checklist
If the database has been modified:
- [ ] Update database diagram 
- [ ] Create new migration 

If API endpoints has been modified:
- [ ] Update backend documentation 
- [ ] Create/Update validators for the API endpoints 
- [ ] The API endpoints are properly secured with keycloak 

If the UI has been modified:
- [ ] It is translated in both language (with no hard coded text) 
- [ ] Has been tested on IE
- [ ] The modifications are tab friendly
- [ ] It is accessible

### CODE REVIEW

**Comment by dynamic11** [comment]
@SukhsimranpreetSekhon can you test this with screen reader
@BenJeau the language button is intentionally red. Once the new "theming" goes it it will fix it

File: services/frontend-v3/src/components/privacyModal/PrivacyModalView.jsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+      width={700}
+    >
+      <div className="privacyModalContent">
+        {locale === "ENGLISH" ? (
+          <>
+            <p>
+              Please be advised that by logging in you have the ability to
+              self-administer information about yourself. The collection of your
+              self-administered information is known as your Profile information
+              to showcase your professional history. You will be able to control
```
Comment (mo-5) [inline services/frontend-v3/src/components/privacyModal/PrivacyModalView.jsx:46]: Just wondering if we should move this to the i18. This way if we ever need to make an update or add a language it would be easier and keeps the code clean

Comment (dynamic11) [reply services/frontend-v3/src/components/privacyModal/PrivacyModalView.jsx:46]: the reason I left it in here for now is that I'm waiting for the final version of the "terms" to see the formatting and how its broken up. once it is finalized i will see how to put it in the translations file. this is essentially a place holder

**Comment by SukhsimranpreetSekhon** [comment]
> @SukhsimranpreetSekhon can you test this with screen reader
> @BenJeau the language button is intentionally red. Once the new "theming" goes it it will fix it

**Comment by SukhsimranpreetSekhon** [comment]
@dynamic11 Sorry I was trying to reply to the comment and I closed the PR by accident (fixed/re-opened it now). The screen reader works perfectly (especially the language button) for everything in the privacy modal in both English and French.

**Review by SukhsimranpreetSekhon — APPROVED** [review]

File: services/frontend-v3/src/components/layouts/appLayout/AppLayoutView.jsx [hunk 2]
```diff
@@ -21,35 +21,37 @@ const AppLayoutView = ({
...
+              )}
+            </Content>
+            <Footer />
+          </Layout>
         </Layout>
       </Layout>
-    </Layout>
+    </>
   );
 };
```
Comment (BenJeau) [inline services/frontend-v3/src/components/layouts/appLayout/AppLayoutView.jsx]: I don't understand the change in this file, is it just because we were planning on putting the modal in the AppLayoutView? (should we revert the changes to this file)

File: services/frontend-v3/src/i18n/en_CA.json [hunk 3]
```diff
@@ -315,6 +315,9 @@
   "profile.visibility.status.public": "Visible to everyone",
   "profile.willing.to.relocate.to": "Willing to relocate to",
   "profile.yes": "Yes",
+  "privacy.modal.header": "Privacy Agreement",
```
Comment (BenJeau) [inline services/frontend-v3/src/i18n/en_CA.json]: To keep consistency with the other titles, Agreement should start with a lower case
```suggestion
  "privacy.modal.header": "Privacy agreement",
```

File: services/frontend-v3/src/routes/Routes.jsx [hunk 4]
```diff
@@ -10,8 +10,13 @@ import {
 import Admin from "./Admin";
 import Secured from "./Secured";
 import ScrollToTop from "./ScrollTopTop";
+import PrivacyModal from "../components/privacyModal/PrivacyModal";
 
-const Routes = () => (
+const Routes = () => {
+
+
+
```
Comment (BenJeau) [inline services/frontend-v3/src/routes/Routes.jsx]: Probably want to remove the superfluous lines/spaces

File: services/frontend-v3/src/routes/Routes.jsx [hunk 5]
```diff
@@ -33,7 +38,12 @@ const Routes = () => (
         render={({ location }) => <Secured location={location} />}
       />
     </Switch>
+    <PrivacyModal />
   </>
-);
+)
+
+
+
```
Comment (BenJeau) [inline services/frontend-v3/src/routes/Routes.jsx]: Probably want to remove the superfluous lines/spaces

File: services/frontend-v3/src/utils/login.js [hunk 6]
```diff
@@ -43,6 +43,7 @@ const profileExist = async (userInfo, axios) => {
       email,
       status,
       signupStep,
+      isPrivacyAccepted: false,
```
Comment (BenJeau) [inline services/frontend-v3/src/utils/login.js]: Should remove this, since it is not doing anything now in the slice

**Review by BenJeau — COMMENTED** [review]
Looks good, just a few comments

**Comment by BenJeau** [comment]
small feature request, @MenaMachado should the user be able to dismiss it once and never see it again? Like maybe have a checkbox saying, "Don't show again"?

File: services/frontend-v3/src/i18n/en_CA.json (hunk 3 above)
Comment (dynamic11) [reply services/frontend-v3/src/i18n/en_CA.json]: done

File: services/frontend-v3/src/routes/Routes.jsx (hunk 5 above)
Comment (dynamic11) [reply services/frontend-v3/src/routes/Routes.jsx]: removed

File: services/frontend-v3/src/utils/login.js (hunk 6 above)
Comment (dynamic11) [reply services/frontend-v3/src/utils/login.js]: removed

File: services/frontend-v3/src/components/layouts/appLayout/AppLayoutView.jsx (hunk 2 above)
Comment (dynamic11) [reply services/frontend-v3/src/components/layouts/appLayout/AppLayoutView.jsx]: resolved

**Review by BenJeau — APPROVED** [review]
LGTM

**Comment by dynamic11** [comment]
> small feature request, @MenaMachado should the user be able to dismiss it once and never see it again? Like maybe have a checkbox saying, "Don't show again"?

just spoke to mena about this and she wants them to see it everytime
